cpp/inc/bond/ext/grpc/detail/service.h

```diff
@@ -35,7 +35,8 @@ template <typename TThreadPool>
 class service : private grpc::Service
 {
 public:
-    virtual ~service() { }
```

Comment:
Since this is a detail type, the client isn't supported to mention it, so there's no way to have something to polymorphically destroy. Good catch.

Reply:
The `grpc::Service` already has a `virtual` destructor, so this is useless.

Reply:
Ah. In that case, can we also add a `has_virtual_destructor` static_assert test against one of the generated `ServiceCore` classes?

The API that has been shipped today explicitly says "You can polymorphically delete things derived from the generated `ServiceCore`.", and such a test will guard against the unlikely case that `grpc::Service` has it's virtual dtor removed.